INN/umbrella-caribbean
# Header updates

## Changes

This pull request makes the following changes:

- Adds new `Header Widget - Right` sidebar for widgets that are supposed to be on the right side of the logo in the header
- Styles that new sidebar so Largo Follow and Search widgets look as expected inside of it
- Decreases size of left and right sidebars so they have the same spacing

Why a new sidebar?
Previously, we were just using a flex order to display the widgets in the `Header Widget` widget area in the correct order. Unfortunately, you can't stack 2/4 flex items, so this is a cleaner solution.

![Screen Shot 2019-12-04 at 3 52 30 PM](https://user-images.githubusercontent.com/18353636/70180516-2f2fbe00-16ae-11ea-8f1b-e054437b3688.png)

## Why



For #69

## Testing/Questions

Features that this PR affects:

- Header


Questions that need to be answered before merging:


Steps to test this PR:

1. Remove the Largo Follow/Social Icon text widget from the Header Widget area
2. Add the Largo Follow widget to the Header Widget - Right area
3. Add the search widget to the Header Widget - Right area
4. View on frontend and verify it looks as expected
